From ac41b401d274a4004027fa4000d801cd28c51f4c Mon Sep 17 00:00:00 2001
From: Alexandru Gagniuc <mr.nuke.me@gmail.com>
Date: Tue, 13 May 2025 13:41:37 -0500
Subject: [PATCH] net: ethernet: qualcomm: ppe: select correct PCS dependency

The config symbol for the PCS driver has changed to PCS_QCOM_IPQ9574,
since the original submission. Update Kconfig accordingly.

Signed-off-by: Alexandru Gagniuc <mr.nuke.me@gmail.com>
---
 drivers/net/ethernet/qualcomm/Kconfig | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/net/ethernet/qualcomm/Kconfig
+++ b/drivers/net/ethernet/qualcomm/Kconfig
@@ -67,7 +67,7 @@ config QCOM_PPE
 	depends on COMMON_CLK
 	select REGMAP_MMIO
 	select PHYLINK
-	select PCS_QCOM_IPQ_UNIPHY
+	select PCS_QCOM_IPQ9574
 	help
 	  This driver supports the Qualcomm Technologies, Inc. packet
 	  process engine (PPE) available with IPQ SoC. The PPE includes
